astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -13,6 +13,8 @@
 
 default_args = {
     "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "retries": int(os.getenv("DEFAULT_TASK_RETRIES", 2)),
```

Comment:
Does it start from the failed task or from the first task when the retry runs?

Reply:
No, retry param you are setting at DAG level so let's say I have three tasks in DAG (A -> B -> C) Now B failed for some reason then retry will resume re-run from task B or task A?

Reply:
retry should be set at task level only